javascript-constructors/new-objects.js

```diff
@@ -0,0 +1,24 @@
+var anObject = new Object();
+console.log('value of anObject:', anObject);
+console.log('typeof anObject:', typeof anObject);
+
+var anArray = new Array();
+console.log('value of anArray:', anArray);
+console.log('typeof anArray:', typeof anArray);
+
+console.log('value of aFunction:', function() {
+
+});
+console.log('typeof aFunction:', typeof function() {});
```

Comment:
Please create a function using the `Function` constructor:

---

![Screenshot_2](https://user-images.githubusercontent.com/16143771/110579925-84157c00-811c-11eb-86cd-ac59e8a9ddc1.jpg)